### PR TITLE
[CP-2996] Implemented search results for search component

### PR DESCRIPTION
**Pull request by mkurczewski**
JIRA Reference: [CP-2996]

### :memo: Description ️

-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-2996]: https://appnroll.atlassian.net/browse/CP-2996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/generic-view/feature/src/lib/seed-data/contacts-seed-data.ts [hunk 1]
```diff
@@ -181,6 +182,16 @@ const contactWithPlusAsSpecialCharacterInFirstName = {
...
 
+const contactWithRepetitions = {
+  firstName: "Marek",
+  lastName: "Marmarecki",
+  phoneNumbers: [
+    { phoneNumber: "500500500", phoneType: "MOBILE" },
+    { phoneNumber: "500500600", phoneType: "WORK" },
+  ],
+  entityType: "contacts",
+}
```
Comment (dkarski) [inline libs/generic-view/feature/src/lib/seed-data/contacts-seed-data.ts:193]: 👍 ❤️

File: libs/generic-view/ui/src/lib/interactive/form/input/search-input.tsx [hunk 2]
```diff
@@ -60,17 +94,45 @@ export const SearchInput: APIFC<FormSearchInputData, FormSearchInputConfig> = ({
...
+            componentRef: resultsRef,
+          })
+        }
+        return null
+      })}
     </Wrapper>
   )
 }
 
 export default SearchInput
```
Comment (dkarski) [inline libs/generic-view/ui/src/lib/interactive/form/input/search-input.tsx:112]:
```suggestion
```

**Review by dkarski — APPROVED** [review]
✅

File: libs/generic-view/ui/src/lib/interactive/form/input/search-input.tsx (hunk 2 above)
Comment (mkurczewski) [reply libs/generic-view/ui/src/lib/interactive/form/input/search-input.tsx]: Removed in [c1d1f9a](https://github.com/mudita/mudita-center/pull/2203/commits/c1d1f9adf8d78a3003a7f9ae66d0ec81eb9ec644)

**Pull request merged**